app/billing/rest.py

```diff
@@ -192,3 +204,17 @@ def update_free_sms_fragment_limit_data(service_id, free_sms_fragment_limit, fin
             free_sms_fragment_limit,
             financial_year_start
         )
+
+
+def serialize_ft_billing(data):
+    results = []
+    no_emails = [x for x in data if x.notification_type != 'email']
+    for d in no_emails:
```

Comment:
Consider rename x, d and j to be more meaningful
x= notification
j = result
d = email?